### ISSUE TITLE
Output to file bug when using a network you have already collected pins.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/reaver-wps#95]
```
What steps will reproduce the problem?

1. Start a cracking session on your network.
2. Ctrl-C out of it, then try to start a session on the same AP using stdout 
(-o).

What is the expected output? What do you see instead?

I would expect either the existing data collected be transferred to the file, 
or at least the file starting new. Currently nothing happens.


What version of the product are you using? On what operating system?
r69


Please provide any additional information below.

Basically, Reaver is outputting "[?] Restore previous session? [n/Y]" to the 
file. I obviously can't select yes or no :) Minor bug, but I thought I'd 
mention it. Thanks for the great tool.



```

Original issue reported on code.google.com by `philippe...@hotmail.com` on 6 Jan 2012 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Thanks, I'll fix this; a temporary work-around is to explicitly specify the 
path to the session file (/etc/reaver/<bssid>.wpc) with --session. It shouldn't 
prompt you then.
```

Original comment by `cheff...@tacnetsol.com` on 6 Jan 2012 at 2:56

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the latest SVN.
```

Original comment by `cheff...@tacnetsol.com` on 6 Jan 2012 at 4:20
- Changed state: **Fixed**
